### DISCUSSION TITLE
Fields displaying one value from another collection.

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#17619] [Feature Requests]
### Summary

Suppose we have 2 collections
collection 1
item, name, group etc
collection 2
item, price, supplier, quantity etc

We create an M2M field by combining both of these collections with an additional field and in the form view in this field we can display all information from collection 2 (this is currently the case in Directus)

But what if we want to have one value in this field, for example price, and we want to have another field in the form where only the quantity (from collection 2) will be displayed. In the table view it is, but it is missing in the form - record tab.

And my idea is a field that uses an already established relationship but only displays the selected information. The functionalities could be extended later with, for example, join-type database functions.

This functionality would work when views will be available and then specific information in specific fields would be displayed from this view database.

### Motivation

If you have fields with a lot of content in the second collection, it doesn't look good like everything is displayed in one field of the first collection. By clicking on this field, you can preview this value, but to display individual values, you could use a field that is not created in the database of this collection, but is an alias to the value from the other one.
Maybe this would allow you to implement join functions etc or improve the display of data from view tables.

### Detailed Design

Additional selection field of the alias type field with the option of choosing what value is to be displayed and what type of value it is in order for it to be displayed correctly

### Requirements List

It should be possible to display in the alias field the values from all the values resulting from the relationship.

### Drawbacks

You will need to somehow display data from view tables or from existing relationships, and it's a good idea to expand on that. This functionality would be most needed for those who use the app to display data as a preview of data in the database. 

### Alternatives

I have no idea

### Adoption Strategy

I have no idea

**Comment by koksikus** [comment]
Is this a stupid idea?  I see a complete lack of interest even to develop a thought.  Is it too early to implement this?  This makes a lot of sense if someone uses directus as a frontend.

**Reply by rijkvanzanten** [reply]
I don't think it's a stupid idea. I see the lack of engagement and upvotes more as a sign that people either don't fully understand the use case, or don't see the same need to have it 🤷🏻

**Comment by koksikus** [comment]
Probably most use directus as intended and additionally create a frontend 😉 However, I use Directus exactly as a frontend for data from the database. I modify the data and preview it and sometimes a field with one value is missing. I know that I can choose the data to be displayed from the relationship in this field and in the table view, its possible, but in the form view it can't be done anymore.  In the future, when will be connected to views, you may want to display one value per field from view. Of course, this is only if someone uses Directus as a frontend. I believe that this can also be a great option to develop and attract users to use this project... 
I'm not a developer and I use it differently than everyone else that's why all my submissions are about the app 😄

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
